### ISSUE TITLE
custom widgets need to support custom names to differentiate them from each other

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#5647]
especially on Android, where they all get labelled precisely the same thing, so you don't know which you're selecting

**Comment by stephanerosa** [comment]
Yes, that would be a very nice quick fix because it does really not make any sense from a UX prospective.

![image](https://user-images.githubusercontent.com/26009192/96690550-1bec1680-1384-11eb-9f51-462911ffab58.png)

**Comment by t3chguy** [comment]
This is a Scalar issue, the Widgets API lets them do that

**Comment by stephanerosa** [comment]
Actually I didn't think of changing that state myself manually. It works like a charm :-)
(Ok DeepL does not load but it was just to test the naming)

![image](https://user-images.githubusercontent.com/26009192/96700046-4394ac00-138f-11eb-8b77-a46b68f3577c.png)

**Comment by t3chguy** [comment]
For the icon you can specify `avatar_url` with an `mxc` resource in the same widget event content

**Comment by stephanerosa** [comment]
That perfectly worked. Thank you !
![image](https://user-images.githubusercontent.com/26009192/96716897-f8d25e80-13a5-11eb-96d5-f178b2605c45.png)

**Comment by t3chguy** [comment]
Hilariously I just realised I never tested `avatar_url` on Widgets when I added support for it, you appear to be its first tester :D

**Comment by stephanerosa** [comment]
I confirm it working on Element-web and Element-desktop.
On Element-android the renaming works but there is no avatar in the first place so I cannot tell
I do not have an iPhone to test Avatar-ios

**Comment by t3chguy** [comment]
The MSC which adds support for avatars on Widgets is new and only Web/Desktop support it yet
